README.md

```diff
@@ -127,10 +127,9 @@ The robust virtual staining models (i.e *VSCyto2D*, *VSCyto3D*, *VSNeuromast*),
 
 A full illustration of the virtual staining pipeline can be found [here](https://github.com/mehta-lab/VisCy/blob/dde3e27482e58a30f7c202e56d89378031180c75/docs/virtual_staining.md).
 
-## DynaCLR (Contrastive learning of representations of cell dynamics)
+## DynaCLR: Contrastive Learning of Cellular Dynamics with Temporal Regularization
```

Comment:
The virtual staining section above is inside parentheses and in sentence case. We should choose one style for both.